bridge/client/app/project-board/project-board.component.ts

```diff
@@ -16,51 +17,57 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
   private readonly unsubscribe$ = new Subject<void>();
 
   public logoInvertedUrl = environment?.config?.logoInvertedUrl;
-  public project$: Observable<Project | undefined>;
+  public projectName$: Observable<string>;
```

Comment:
Do we need this as a global variable?

Reply:
No, you are right. Also a leftover from testing tryouts.  I will remove it.